src/app/static/src/tests/components/runWorkflow/runWorkflowProgress.test.ts

```diff
@@ -88,6 +88,15 @@ describe(`runWorkflowProgress`, () => {
         })
     })
 
+    it(`initial selected workflow is set by props and emitted`, async (done) => {
```

Comment:
Could also include a test that event is emitted when selected workflow changes from within the component, not prop on mount